editors/emacs/files/apple-silicon-devel.diff

```diff
@@ -0,0 +1,10 @@
+--- configure.ac.orig	2020-09-06 11:42:39.000000000 -0400
++++ configure.ac	2020-09-06 11:43:22.000000000 -0400
+@@ -709,6 +709,7 @@
+     case "${canonical}" in
+       *-apple-darwin[0-9].*) unported=yes ;;
+       i[3456]86-* | x86_64-* )  ;;
++      aarch64-* )  ;;
```

Comment:
Thinking that this added line and the one for the release should match ... yes? Right now they don't ...

Reply:
I updated the patch files to check for both possible target values in both the ports.